src/swagger.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
I guess it could go into src/swagger-docs/ or something? Wasn't sure where to put it really